src/varity/vcf_to_hgvs/protein.clj

```diff
@@ -231,13 +253,14 @@
         [_ _ offset _] (diff-bases pref palt)
         ref (nth (:ref-prot-seq seq-info) (dec (+ ppos offset)))
         alt (nth (:alt-prot-seq seq-info) (dec (+ ppos offset)))
-        ter-site (-> (:alt-prot-seq seq-info)
+        ter-site (-> seq-info
+                     format-alt-prot-seq
                      (subs (dec (+ ppos offset)))
                      (string/index-of "*"))]
     (mut/protein-frame-shift (mut/->long-amino-acid ref)
                              (coord/protein-coordinate (+ ppos offset))
                              (mut/->long-amino-acid alt)
-                             (if ter-site
+                             (if (and ter-site (< 1 ter-site))
```

Comment:
According to hgvs.org, `fs*1` should be described as a nonsense mutation.
(e.g. `E746*`)
Current implementation translates such variants to frame shift mutations.
(e.g. `E746*fs*?`)

We should fix this translation issue eventually. L139 in `vcf_to_hgvs_test.clj` is a test case for this.

cf .http://varnomen.hgvs.org/recommendations/protein/variant/frameshift/

> the description of a frame shift variant can not contain “fsTer1”, such a variant is a nonsense variant (see Substitution). The shortest frame shift variant possible contains “fsTer2”